labelbox/schema/role.py

```diff
@@ -0,0 +1,41 @@
+from dataclasses import dataclass
+
+from labelbox.orm.model import Field
+from labelbox.orm.db_object import DbObject
+from labelbox.schema.project import Project
+
+_ROLES = None
+
+
+def get_roles(client):
+    global _ROLES
+    if _ROLES is None:
+        query_str = """query GetAvailableUserRolesPyApi { roles { id name } }"""
+        res = client.execute(query_str)
+        _ROLES = {}
+        for role in res['roles']:
+            role['name'] = format_role(role['name'])
+            _ROLES[role['name']] = Role(client, role)
+    return _ROLES
+
+
+def format_role(name: str):
```

Comment:
jw, do we even need to format_roles now that we are not emulating enum behaavior?

Reply:
Idk I don't like spaces and inconsistent casing in key names. It makes it harder to use.